xtask/src/main.rs

```diff
@@ -174,6 +175,26 @@ pub fn run_cargo_fuzz(opt: &RunCargoFuzz) -> Step {
     }
 }
 
+pub fn run_cargo_vet() -> Step {
+    let all_affected_crates = all_affected_crates(&Scope::DiffToMain);
+    Step::Multiple {
+        name: "cargo vet".to_string(),
+        steps: crate_manifest_files()
```

Comment:
What is the plan if we want to enable it more widely in future? It seems that once this is a required CI check we will have to vet all the dependencies for a workspace before we can add `supply-chain/config.toml` to that workspace.

Reply:
By default when you run `cargo vet init`, it'll mark all the dependencies to be ignored, so the test wouldn't fail.

I explicitly didn't mark the dependencies as ignored when first setting up `cargo vet` for stage0 as we didn't have a test, so it didn't matter that `cargo vet` fails.